### PR TITLE
GitHub actions

### PR DESCRIPTION
**Pull request by dtebbs**
Switch from travis to github actions.
Addresses #164.

### CODE REVIEW

**Comment by dtebbs** [comment]
Seems to be working now, so removing WIP.

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,66 @@
+# This is a basic workflow to help you get started with Actions
```
Comment (AntoineRondelet) [inline .github/workflows/main.yml]: If that from a template you have used? maybe we can remove this line

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+name: CI
```
Comment (AntoineRondelet) [inline .github/workflows/main.yml]: What about `zeth-ci` instead?

File: .github/workflows/main.yml [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+# events but only for the master branch
+on:
+  push:
+  pull_request:
```
Comment (AntoineRondelet) [inline .github/workflows/main.yml]: Woudn't it be possible to run the ci also on pushes on develop by specifying `master` and `develop` here ? (https://help.github.com/en/actions/configuring-and-managing-workflows/configuring-a-workflow#filtering-for-specific-branches-tags-and-paths)

File: .github/workflows/main.yml (hunk 1 above)
Comment (dtebbs) [reply .github/workflows/main.yml]: Removed the comments.

File: .github/workflows/main.yml (hunk 2 above)
Comment (dtebbs) [reply .github/workflows/main.yml]: Changed to `zeth-ci`

File: .github/workflows/main.yml (hunk 3 above)
Comment (dtebbs) [reply .github/workflows/main.yml]: As far as I know, this *will* run the ci on pushes to master and develop (once this is merged).  It should work on any branch.  If we add a filter then I think it *only* runs on those branches.
Let me know if you think I've missed something.

Comment (AntoineRondelet) [reply .github/workflows/main.yml]: Ah right. Thanks

**Comment by AntoineRondelet** [comment]
Thanks @dtebbs LGTM

**Pull request merged**